### PR TITLE
Fix the selected color scheme list item container background being blue in Windows 10

### PR DESCRIPTION
**Pull request by PankajBhojwani**
Specify the resource to use for the list view item background when selected.

References #14693 

## Validation Steps Performed
Background is no longer blue, it is light gray (like in Windows 11). Screenshots below.

### CODE REVIEW

**Comment by PankajBhojwani** [comment]
Win 11:
image

**Review by zadjii-msft — APPROVED** [review]

File: src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml [hunk 1]
```diff
@@ -70,6 +70,9 @@
                   PreviewKeyDown="ListView_PreviewKeyDown"
                   SelectedItem="{x:Bind ViewModel.CurrentScheme, Mode=TwoWay}"
                   SelectionMode="Single">
+            <ListView.Resources>
+                <SolidColorBrush x:Key="ListViewItemBackgroundSelected" Color="{ThemeResource SystemListLowColor}" />
```
Comment (carlos-zamora) [inline src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml:74]: nit: I would mention that we need to do this because of Win10

**Review by carlos-zamora — APPROVED** [review]

**Review by DHowett — CHANGES_REQUESTED** [review]
What does it look like on Windows 10?

**Comment by PankajBhojwani** [comment]
Win 10:
image

**Review by DHowett — APPROVED** [review]

**Comment by DHowett** [comment]
@msftbot merge this in 5 minutes

**Pull request merged**